test/bigtest/interactors/order-edit-page.js

```diff
@@ -27,10 +27,15 @@ import { TIMEOUT } from './const';
   template = new Button('[name="template"]');
 }
 
+@interactor class OngoingInfoAccordion {
+  static defaultScope = '#ongoing';
+  isSubscriptionCheckbox = clickable('[data-test-checkbox] label');
```

Comment:
OK, that looks better, but here is the thin line - `clickable` makes the value as function. And maybe it's better to call attribute like `clickIsSubscriptionCheckbox` or something. In case you use `isSubscriptionCheckbox = new CheckboxInteractor('#some-id');` - your name works perfectly. I hope now difference is clear. It can go with current way, however naming of vars is pretty important, since we write code for humans, not machines :smile_cat: